operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
```

Comment:
Do we have to remove this now? In mimir release we remove main/unreleased when bumping up the version.

Reply:
good point